package.json

```diff
@@ -87,7 +87,7 @@
     "@mui/internal-scripts": "^1.0.3",
     "@mui/internal-test-utils": "https://pkg.csb.dev/mui/material-ui/commit/fb183624/@mui/internal-test-utils",
     "@mui/material": "5.15.15",
-    "@mui/monorepo": "github:mui/material-ui#81037854cccccb7ff20fbfebed4334585babab0e",
+    "@mui/monorepo": "github:mui/material-ui#bbfb2cd742ed3aca3d0c8f6dc3e890ed4fc1d7f2",
```

Comment:
Should be

```suggestion
    "@mui/monorepo": "github:mui/material-ui#next",
```

otherwise Renovate (once we configure it here) will try to use the default branch.